source/index.html.haml

```diff
@@ -115,13 +114,18 @@ title: Erie Day of Code | Sign up for updates!
                 #{a.address}
             %p.phone= a.phone
 
-    %p If none of these accommodations suit you, please visit #{link_to('Visit PA', 'http://www.visiteriepa.com/where-to-stay')}, or contact #{mail_to('contact', 'contact@eriedayofcode.com')} us to help make arrangements!
+    %p If none of these accommodations suit you, #{link_to('Visit PA', 'http://www.visiteriepa.com/where-to-stay')}, or email #{mail_to('contact', 'contact@eriedayofcode.com')} and we'll help you make arrangements!
```

Comment:
I actually thought that it was fun to have it without anything. It reads nice... Visit PA linked is quite obvious IMO.